tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -366,10 +384,19 @@ protected override bool RemoveFast(string key)
             return MaybeUnknown?.Remove(key) ?? false;
         }}
         protected override void ClearFast()
-        {{
-            _bits = 0;
-            _headers = default(HeaderReferences);
+        {{            
             MaybeUnknown?.Clear();
+            {Each(loop.Headers.OrderBy(h => !h.PrimaryHeader), header => $@"
+                if ({header.TestBit()})
+                {{
+                    _headers._{header.Identifier} = default(StringValues);
```

Comment:
The HeaderReferences is a pretty big struct - does the `= default()` change improve after this tweak in coreclr? https://github.com/dotnet/coreclr/pull/7198

Alas its not currently part of 1.1.0; was going to be added https://github.com/dotnet/coreclr/pull/7341 but..?